base_tier_validation/models/tier_validation.py

```diff
@@ -35,11 +35,26 @@ class TierValidation(models.AbstractModel):
         search="_search_reviewer_ids",
     )
     can_review = fields.Boolean(compute="_compute_can_review")
+    has_comment = fields.Boolean(
+        related='review_ids.has_comment',
```

Comment:
If review_ids is multi records, I am not so sure this will has singleton error.
Can you check and confirm?

Reply:
I will change to compute field like can_review.